content/docs/istio/cr-istio.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   size: 1
   image: vault:1.3.1
-  bankVaultsImage: banzaicloud/bank-vaults:master
+  bankVaultsImage: banzaicloud/bank-vaults:latest
```

Comment:
The correct image now is `ghcr.io/bank-vaults/bank-vaults`

Reply:
Updated, thanks! I'll also create a separate PR to update the same in https://github.com/bank-vaults/vault-operator